op-supervisor/supervisor/types/types.go

```diff
@@ -352,3 +352,48 @@ type ManagedEvent struct {
 	DerivationUpdate *DerivedBlockRefPair `json:"derivationUpdate,omitempty"`
 	ExhaustL1        *DerivedBlockRefPair `json:"exhaustL1,omitempty"`
 }
+
+type SuperRootResponse struct {
+	Timestamp uint64 `json:"timestamp"`
```

Comment:
nit: I think I forgot this in my original draft, but we should include the actual super root (common.Hash) and probably its preimage (eth.SuperV1)

Reply:
Not entirely sure about the eth.SuperV1 actually - we have the Canonical below already so we're only missing the chain IDs.  Maybe we should just add those in here too?

Reply:
yeah the challenger should be able to reconstruct the super root from the canonicals. I'll just add the chain IDs then.

Reply:
The hash would still be very useful for dispute-mon and op-proposer though, but agreed we don't need the full pre-image and can leave challenger to reconstruct that.

Reply:
It's fixed. I've added the super root to the response.